src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -736,6 +782,29 @@ public void isSentToMultiSig_whenNonStandardErpRedeemScript_shouldReturnTrue() {
         Assert.assertTrue(nonStandardErpRedeemScript.isSentToMultiSig());
     }
 
+    @Test
+    public void isSentToMultiSig_whenP2shErpRedeemScript_shouldReturnTrue() {
+        Script p2shErpRedeemScript = createP2shErpRedeemScript(
+            FEDERATION_KEYS,
+            ERP_FEDERATION_KEYS,
+            CSV_VALUE
+        );
+
+        Assert.assertTrue(p2shErpRedeemScript.isSentToMultiSig());
+    }
+
+    @Test
+    public void isSentToMultiSig_whenFlyoverStandardMultiSig_shouldReturnTrue() {
```

Comment:
```suggestion
    public void isSentToMultiSig_whenFlyoverStandardRedeemScript_shouldReturnTrue() {
```

just to be consistent with other flyover test names

Reply:
Updated